src/main/java/uk/gov/companieshouse/exemptions/delta/Config.java

```diff
@@ -38,11 +39,13 @@ public class Config {
     public ConsumerFactory<String, ChsDelta> consumerFactory(@Value("${spring.kafka.bootstrap-servers}") String bootstrapServers) {
         return new DefaultKafkaConsumerFactory<>(new HashMap<>() {{
             put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
-            put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
-            put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ChsDeltaDeserialiser.class);
+            put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, ErrorHandlingDeserializer.class);
```

Comment:
Are these config changes necessary?

Reply:
Re-tested without config changes; makes the application loop again.
Conclusion is that these config changes are required for fix to work.